services/slack/intermediate.go

```diff
@@ -384,6 +384,23 @@ func (t *Transformer) CreateIntermediateUser(user string) {
 	t.Logger.Warnf("Created a new user because the original user was missing from the import files. user=%s", user)
 }
 
+func (t *Transformer) CreateAndAddPostsToThreads(post SlackPost, threads map[string]*IntermediatePost, timestamps map[int64]bool, channel *IntermediateChannel) {
+	author := t.Intermediate.UsersById[post.User]
+	if author == nil {
+		t.CreateIntermediateUser(post.User)
+		author = t.Intermediate.UsersById[post.User]
+	}
+
+	newPost := &IntermediatePost{
+		User:     author.Username,
+		Channel:  channel.Name,
+		Message:  post.Text,
+		CreateAt: SlackConvertTimeStamp(post.TimeStamp),
```

Comment:
Shouldn't this be `CreatedAt`?

Reply:
the struct is already defined and thats why we haven't changed it